doc/source/rest.j2

```diff
@@ -551,15 +551,134 @@ argument, and in this case the second argument passed is the value, or it.
 The operators or (`or` or `∨`), and (`and` or `∧`) and `not` are also
 supported, and take a list of arguments as parameters.
 
+.. _aggregates:
+
+Aggregates: on the fly, measurements modification and aggregation
+===================================================================
+
+Gnocchi allows to do on-the-fly aggregation and modification of already
+aggregated data of |metrics|.
+
+It can be done by providing the list of |metrics| to aggregate:
+
+{{ scenarios['get-aggregates-by-metric-ids']['doc'] }}
+
+This example computes the mean aggregates with `all` metrics listed in
+`metrics` and then multiples it by `4`.
+
+.. Note:
+
+   `operations` can also be passed as a string, for example:
+   `"operations": "(aggregate mean (metric all))"`
```

Comment:
I'd prefer break a line and using code format:
```
`operations` can also be passed as a string, for example:
:code:`"operations": "(aggregate mean (metric all))"`
```

And what is `all` here means? I cannot find explanation in the doc